#ifndef MAINWINDOW_H
#define MAINWINDOW_H

#include "../libqmsstyle.h"
#include "../style/style.h"

#include <QMainWindow>

#include <QProcess>

QT_BEGIN_NAMESPACE
namespace Ui {
class MainWindow;
}
QT_END_NAMESPACE

class MainWindow : public QMainWindow
{
    Q_OBJECT

public:
    MainWindow(QWidget *parent = nullptr);
    ~MainWindow();

public slots:
    void readMsstyle_clicked();

private:
    Ui::MainWindow *ui;

    LibQmsstyle *m_loadedMsstyle;
};
#endif // MAINWINDOW_H
